overlord/fdestate/fdemgr_test.go

```diff
@@ -132,6 +132,9 @@ func (s *fdeMgrSuite) startedManager(c *C) *fdestate.FDEManager {
 }
 
 func (s *fdeMgrSuite) TestGetManagerFromState(c *C) {
+	if !secboot.WithSecbootSupport {
+		c.Skip("secboot is not available")
+	}
```

Comment:
can't you use the build tag on the test file itself? other option is to do the skip from a SetUpSuite method

Reply:
I did not realize all the tests were skipped.